docs/developer/plugin-list.asciidoc

```diff
@@ -392,6 +392,10 @@ actitivies.
 |The features plugin enhance Kibana with a per-feature privilege system.
 
 
+|{kib-repo}blob/{branch}/x-pack/plugins/file_data_visualizer[fileDataVisualizer]
```

Comment:
Should add the `file_data_visualizer` plugin to the ML section of the `CODEOWNERS` file.